ci/run_envoy_docker.sh

```diff
@@ -38,14 +38,14 @@ else
   ENVOY_DOCKER_OPTIONS+=(-v /var/run/docker.sock:/var/run/docker.sock)
   ENVOY_DOCKER_OPTIONS+=(--cap-add SYS_PTRACE --cap-add NET_RAW --cap-add NET_ADMIN)
   DEFAULT_ENVOY_DOCKER_BUILD_DIR=/tmp/envoy-docker-build
-  BUILD_DIR_MOUNT_DEST=/build
+  [[ -z "${BUILD_DIR_MOUNT_DEST}" ]] && BUILD_DIR_MOUNT_DEST="/build" # Salvo unique
   SOURCE_DIR="${PWD}"
-  SOURCE_DIR_MOUNT_DEST=/source
+  [[ -z "${SOURCE_DIR_MOUNT_DEST}" ]] && SOURCE_DIR_MOUNT_DEST="/source" # Salvo unique
   START_COMMAND=("/bin/bash" "-lc" "groupadd --gid $(id -g) -f envoygroup \
-    && useradd -o --uid $(id -u) --gid $(id -g) --no-create-home --home-dir /build envoybuild \
+    && useradd -o --uid $(id -u) --gid $(id -g) --no-create-home --home-dir ${BUILD_DIR_MOUNT_DEST} envoybuild \
```

Comment:
The comment `# Salvo unique` is required in each changed line.

Reply:
add comment behind a backslash for each line  in a multiline command will break this command. I take an alternative way  as below 
```
echo CommandName InputFiles `#1st comment` \
             --option1 arg1 `#2nd comment` \
             --option2 arg2 `#3rd comment`
```
it's not align with other comment style but it's the only way i found by now.

Reply:
It works for me, the comment `#Salvo unique` is important for maintainers to figure out the difference from Nighthawk and Envoy scripts.